front/src/reducers/map/index.ts

```diff
@@ -34,17 +34,16 @@ export interface MapState {
   smoothTravel: boolean;
   viewport: Viewport;
   layersSettings: {
```

Comment:
Use layersSettings here
```suggestion
  layersSettings: LayersSettings
```